test/system/410-selinux.bats

```diff
@@ -239,12 +239,20 @@ function check_label() {
     run ls -dZ $tmpdir
     is "$output" "${RELABEL} $tmpdir" "Privileged Relabel Correctly"
 
-    run_podman run -v $tmpdir:/test:Z $IMAGE cat /proc/self/attr/current
+    run_podman run --name label -v $tmpdir:/test:Z $IMAGE cat /proc/self/attr/current
     level=$(secon -l $output)
     run ls -dZ $tmpdir
     is "$output" "system_u:object_r:container_file_t:$level $tmpdir" \
        "Confined Relabel Correctly"
 
+    chcon -t usr_t -l s0 $tmpdir
+    run_podman start --attach label
+    newlevel=$(secon -l $output)
+    is "$level" "$newlevel" "start should use same SELinux labels"
+    run ls -dZ $tmpdir
+    is "$output" "system_u:object_r:usr_t:s0 $tmpdir" \
+       "Start did not Relabel"
+
```

Comment:
Can you add a test for the chown change too?  It would be nice to have a note in the description about that change.